Comments/Migrations/20210421145620_SubmittedToLeadCountsView.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Comments.Migrations
+{
+    public partial class SubmittedToLeadCountsView : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql($@"
```

Comment:
this seems to be a copy of SubmittedCommentAndAnswerCount, except with the organisation id added to the select list, and the condition:
"WHERE s.RespondingAsOrganisation = 1" added.

Do you think it'd be worth just having 1 view, which can have the organisation id in it (which would be null for submissions not to a lead), and could have the RespondingAsOrganisation in the select list?

then the code that calls it can just add the RespondingAsOrganisation condition.

Feel free to disagree though if you think there's value to having the extra view. it's just an idea/suggestion.